tests/results/24a70216d2914871c4c1f0f4a7aa5f8fc1c6d0e1b2c2718f62ade6903680aaeb_slashdot/result.json

```diff
@@ -3,24 +3,37 @@
     "drop_file": false,
     "score": 0,
     "sections": [
+      {
+        "auto_collapse": false,
+        "body": "URL 'https://news.slashdot.org/comments.pl?sid=11253499&cid=55399795' (380369f835d00015f089274107cf114d) was requested but a different URL was fetched (d86e2b62b352d8f9c944980f534f86fe).",
```

Comment:
Can you check all results.json where they have changed? They should not have changed unless you meant to change it like the sample_zip test where you modified the IP address.